New pages: HTMLInputElement step, name, and defaultValue properties (#35614)
mdn/content: files/en-us/web/api/htmlinputelement/name/index.md

---
title: "HTMLInputElement: name property"
short-title: name
slug: Web/API/HTMLInputElement/name
page-type: web-api-instance-property
browser-compat: api.HTMLInputElement.name
---

{{ApiRef("HTML DOM")}}

> The **`name`** property of the {{domxref("HTMLInputElement")}} interface indicates the name of the {{HTMLElement("input")}} element, generally defined set by the element's [`name`](/en-US/docs/Web/HTML/Element/input#name) attribute. This attribute can also be provide an element with a different name.

Reviewer: ```suggestion
The **`name`** property of the {{domxref("HTMLInputElement")}} interface indicates the name of the {{HTMLElement("input")}} element. It reflects the element's [`name`](/en-US/docs/Web/HTML/Element/input#name) attribute.
```
